ksp_plugin/plugin.hpp

```diff
@@ -203,12 +203,15 @@ class Plugin {
   // Calls |MakeSingleton| for all parts in loaded vessels, enabling the use of
   // union-find for pile up construction.  This must be called after the calls
   // to |IncrementPartIntrinsicForce|, and before the calls to
-  // |ReportCollision|.
+  // |ReportPartCollision|.
   virtual void PrepareToReportCollisions();
 
-  // Notifies |this| that the given vessels are touching, and should gravitate
+  // Notifies |this| that the given part is touching the ground.
+  virtual void ReportGroundCollision(PartId part) const;
+
+  // Notifies |this| that the given parts are touching, and should gravitate
   // as part of a single rigid body.
-  virtual void ReportCollision(PartId part1, PartId part2) const;
+  virtual void ReportPartCollision(PartId part1, PartId part2) const;
 
   // Destroys the vessels for which |InsertOrKeepVessel| has not been called
```

Comment:
Also the vessels which transitively touch the ground.

Reply:
Done.